addons/binding/org.openhab.binding.upb/.classpath

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The about.html file seems missing, [guidelines B.2](http://docs.openhab.org/developers/development/guidelines.html#b-osgi-bundles)